src/components/Teaser/Teaser.tsx

```diff
@@ -36,12 +36,17 @@ export const Teaser: React.FC<Props> = ({
     ...rest
 }) => {
     const subTitleElement = (
-        <Text inline title={subTitle} context={disabled ? 'muted' : 'accent'} {...elem('subTitle')}>
+        <Text
+            inline
+            title={subTitle}
+            context={disabled ? 'neutral' : 'background'}
```

Comment:
I think this is a mistake. Shouldn't it change to `{disabled ? 'neutral' : 'default'}`?
2 resons:
1. `context: background` is not something we normally have
2. text with background color will basically be invisible